docs/index.d.ts

```diff
@@ -55,6 +55,29 @@ export declare interface Tracer extends opentracing.Tracer {
    * Returns a reference to the current scope.
    */
   scope(): Scope;
+
+  /**
+   * Instrument a function by automatically creating a span and activating the
```

Comment:
It might be easier to digest reading it as:
```js
// Instruments a function by automatically creating an activated span on its scope
```

Reply:
Rephrased it with the suggestion, but slightly different. Let me know if that works.